File: 2137-final-value-of-variable-after-performing-operations/final-value-of-variable-after-performing-operations.cpp
class Solution {
public:
    int finalValueAfterOperations(vector<string>& operations) {
        int X = 0;

        for (const string& op : operations) {
            if (op[1] == '+') {
                X++;
            } else {
                X--;
            }
        }

        return X;
    }
};

File: 2137-final-value-of-variable-after-performing-operations/final-value-of-variable-after-performing-operations.js
/**
 * @param {string[]} operations
 * @return {number}
 */
var finalValueAfterOperations = function(operations) {
    let x = 0;

    for (let op of operations) {
        if (op === "++X" || op === "X++") {
            x++;
        } else if (op === "--X" || op === "X--") {
            x--;
        }
    }

    return x;
};
